ietf/static/css/ietf.scss

```diff
@@ -236,9 +236,12 @@ th,
 
 // Helper to constrain the size of the main logo
 .ietflogo {
-    width: 75%;
```

Comment:
@larseggert Where did this .75 originally come from?

Reply:
No idea. But that used to look correct with the old CSS. 🤷